ui/components/app/transaction-breakdown/transaction-breakdown.container.js

```diff
@@ -2,20 +2,31 @@ import { connect } from 'react-redux';
 import { getShouldShowFiat } from '../../../selectors';
 import { getNativeCurrency } from '../../../ducks/metamask/metamask';
 import { getHexGasTotal } from '../../../helpers/utils/confirm-tx.util';
+import { subtractHexes } from '../../../helpers/utils/conversions.util';
 import { sumHexes } from '../../../helpers/utils/transactions.util';
 import TransactionBreakdown from './transaction-breakdown.component';
 
 const mapStateToProps = (state, ownProps) => {
   const { transaction, isTokenApprove } = ownProps;
   const {
     txParams: { gas, gasPrice, value } = {},
-    txReceipt: { gasUsed } = {},
+    txReceipt: { gasUsed, effectiveGasPrice } = {},
+    baseFeePerGas,
   } = transaction;
 
   const gasLimit = typeof gasUsed === 'string' ? gasUsed : gas;
 
+  const priorityFee =
+    effectiveGasPrice &&
+    baseFeePerGas &&
+    subtractHexes(effectiveGasPrice, baseFeePerGas);
+
+  const usedGasPrice = gasPrice || effectiveGasPrice;
```

Comment:
@danjm Could we add a brief comment explaining this? This is setting this variable according to whether its a legacy or post London (EIP1559) type transaction right? I think its a bit extra confusing because the field name gasPrice isn't used in the post-London transactions right?

Reply:
Done in effectiveGasPrice